Source/Fuse.Nodes/NodeGroup.uno

```diff
@@ -30,9 +35,17 @@ namespace Fuse
 				return _nodes;
 			}
 		}
+
+		TemplateSourceImpl _templates;
+		
+		[UXContent]
+		public IList<Template> Templates { get { return _templates.Templates; } }
+		public Template FindTemplate(string key) { return _templates.FindTemplate(key); }
 		
 		internal NodeGroupBase() { }
 		
+		protected bool UseTemplates = true;
```

Comment:
Maybe you should mark this `readonly` and pass the value through the constructor above instead?

Something like:

```uno
readonly bool _useTemplates;
internal NodeGroupBase(bool useTemplates)
{
	_useTemplates = useTemplates;
}
```

That way it's clear to the reader that this can't change during the lifetime of the instance. It should also allow us to keep the actual field private.